Automattic/jetpack
# Masterbar: add Tracks instrumentation

Fixes #7738
~~Depends on D7169-code~~

#### Changes proposed in this Pull Request:

~~Enqueues tracks scripts when the WordPress.com masterbar is loaded, to make them available to the javascript overrides.~~

Makes the Masterbar and the Notifications Panel interactions trackable by making the record event function global on `tracks-ajax.js` and adding event handling to the Masterbar plugin.

#### Testing instructions:

* ~~Sandbox `s0.wp.com`.~~
* ~~Apply D7169-code on your sandbox.~~
* ~~Using a http debugger, click on any menu item (this does not work on top level items). An entry for the corresponding event with the domain `pixel.wp.com` should appear.~~

* Run a WordPress instance with this branch
* Enable WordPress.com Masterbar
* Using an http debugger, any activity on the masterbar, or opening a site, post or comment on the notifications panel should trigger an `admin-ajax.php` request with the `jetpack_tracks` action.
